coordinator/src/services/handlers/spine-handler.ts

```diff
@@ -81,14 +81,20 @@ export class LiveRequestHandler implements RequestHandler {
     }
   }
 
+  private static handlePotentialFailureCaseForDirectResponse(data: string | undefined): SpineDirectResponse<string> {
+    // If the body contains the magic tag, this represents an error and we should return a 400
+    const isSuccess = data && data.includes("acknowledgement typeCode=\"AA\"")
```

Comment:
think its fairly normal to use single quotes to avoid having to escape double quotes in strings (and pretty sure we do this in other places)